apps/backend/src/deposits/pos-deposit.service.ts

```diff
@@ -92,4 +93,22 @@ export class PosDepositService {
     depositEntity.status = MatchStatus.EXCEPTION;
     return await this.posDepositRepo.save(depositEntity);
   }
+
+  async adjustAmountsForRefundTransactions() {
```

Comment:
This logic should be in the parser.

Reply:
This is just for the current data that is already in the DB, since we've already deployed our database